tests/cases/conformance/salsa/multipleDeclarations.ts

```diff
@@ -7,4 +7,19 @@ function C() {
 }
 C.prototype.m = function() {
     this.nothing();
-};
+}
+
```

Comment:
Need a test that does this in the opposite order (with the constructor below the method declarations)